README-0.15.md

```diff
@@ -0,0 +1,674 @@
+<p align="center">
```

Comment:
is this file supposed to be here in this PR?

Reply:
We use versioned READMEs now, but I forgot to create it during the 0.14 version bump. (This PR adds new generator flags which shows up in the README.)